site/docs/concepts/config-policies.md

```diff
@@ -135,3 +136,18 @@ config[{"copy": decision}] {
 }
 
 ```
+
+### How to add / modify / delete policies
```

Comment:
If we are doing lifecycle management of policies now, we need to add versions to policies.

Reply:
Added a global file with the version value, plus instructions to increase version upon each update.